account/src/main/java/com/caldevsupplychain/account/controller/AccountControllerImpl.java

```diff
@@ -59,11 +61,11 @@
 	/************************************************************************************************
 	 |									Account API													|
 	 ************************************************************************************************/
-	@GetMapping("/users")
-	public ResponseEntity<?> getUsers() {
-		// custommize logic to pass in page search limit
-		List<UserBean> users = accountService.getAllUsers();
-		return new ResponseEntity<Object>(userMapper.userBeansToUserWSs(users), HttpStatus.OK);
+	@GetMapping("/demo")
+	@RequiresAuthentication
+	@RequiresJwtAuthentication
+	public String demo(){
```

Comment:
Can you remove the demo method? The code merging to master should be "clean"

Reply:
fix